airbyte-proxy/build.gradle

```diff
@@ -0,0 +1,21 @@
+plugins {
+    id 'application'
+}
```

Comment:
I believe we can actually get rid of this since this project isn't a java project and the main thing we are doing here is building a docker image. Everything else should continue working.